src/System.IO.FileSystem.Watcher.Polling/System/IO/PollingFileSystemEventHandler.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace System.IO.FileSystem
```

Comment:
I don't think a separate file is necessary for this. It can go in PollingFileSystemEventArgs.cs.

Reply:
Our standard is one class per file isn't it?

Reply:
> Sure, but should we be treating a delegate as a class?

That was my understanding. It isn't about how complex something is, it is having the top level types in their own files. Easier to track/navigate.